src/FakeItEasy.Analyzer.Shared/ArgumentConstraintAnalyzerBase.cs

```diff
@@ -12,7 +12,7 @@ namespace FakeItEasy.Analyzer
     using Microsoft.CodeAnalysis.VisualBasic.Syntax;
 #endif
 
-    public abstract class ArgumentConstraintAnalyzerBase : DiagnosticAnalyzer
+    internal abstract class ArgumentConstraintAnalyzerBase : DiagnosticAnalyzer

```

Comment:
Other analyzers were internal, so I figured why not?

Reply:
Hmm. I only have 2017 at work, and it seems that _none_ of the analyzers work in 5.5.0. When I roll back to 4.9.2, `NonVirtualSetupAnalyzer`, which is internal works, as does `ArgumentConstraintOutsideCallSpecAnalyzer`, which is public.

Reply:
That's annoying... So, it doesn't matter that they're internal, but we must have broken something at some point. I'll try with VS2019.

Reply:
Works in VS2019.
We upgraded the analyzers to .NET Standard 2.0 in 5.4.0 (#1667); that's probably the cause. I didn't expect that they would stop working in VS2017... Well, people stuck on VS2017 can keep using an older version, I guess. But that means we probably should decouple the analyzer version from the library version...